spec/models/link_report_spec.rb

```diff
@@ -29,7 +29,8 @@
           "https://www.gov.uk/also/good/stuff",
           "https://www.gov.uk/not/as/great",
           "http://example.com/good"
-        ]
+        ],
+        webhook_uri: "https://collections-publisher.test.gov.uk/link_report"
```

Comment:
You could move this into its own test, and assert that the correct webhook is passed to link-checker-api.